content/webapp/views/pages/concepts/concept/concept.helpers.ts

```diff
@@ -42,3 +42,251 @@ export const getThemeSectionHeading = (
 
   return `${tabLabel} this ${conceptTypeLabel}`;
 };
+
+type ConceptSection = {
+  display: boolean;
+  label?: string;
+};
+
+type ConceptConfig = {
+  [key: string]: {
+    displayName?: string;
+    showPartOf: boolean;
+    sourcedDescription: boolean;
+    fieldOrArea: ConceptSection;
+    imagesBy: ConceptSection;
+    imagesAbout?: ConceptSection;
```

Comment:
do we need to add `imagesIn` ("images using [genre]")

Reply:
I don't know – how would I determine that? It's not clear to me from the Notion page

Reply:
Just noticing it's also missing from our developers.wc.org documentation... 
Just figured it was there based on the queries in concept pages. @kenoir ?

Reply:
Yes I believe so... Bit annoying to have to spread it everywhere but 🤷‍♀️ unsure what else to do